demos/vite-demo/vite.config.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'path';
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import commonjs from 'vite-plugin-commonjs';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [
+    react({
+      babel: {
+        babelrc: true,
+      },
+    }),
+    commonjs({
+      advanced: {
+        importRules: 'merge',
+      },
+    }),
+  ],
+  resolve: {
+    alias: {
+      '@iso': path.resolve(__dirname, './src/components/__isograph/iso.ts'),
+      '@iso/*': path.resolve(__dirname, './src/components/__isograph/*'),
+    },
+  },
```

Comment:
This will make sure, `vite` transpiles isograph from `commonjs` to `esm` with `esbuild`.
```suggestion
  },
  optimizeDeps: {
    include: ['@isograph/react'],
  },
```

Reply:
Done, worked great. Thank you!